modules/core/src/test/scala/dk/cwconsult/peregrin/core/test/TempgresConnection.scala

```diff
@@ -0,0 +1,42 @@
+package dk.cwconsult.peregrin.core.test
+
+import java.util.UUID
+
+import dk.cwconsult.tempgres.TempgresClient
+import scalikejdbc.ConnectionPool
+import scalikejdbc.GlobalSettings
+import scalikejdbc.LoggingSQLAndTimeSettings
+
+object TempgresConnection {
```

Comment:
Jeg kunne rigtig godt tænke mig at se nogen af de her mere "flyt kode ud af test-klasser" ting puttet i en separat commit (først) sådan at vi bare kan merge dem med det samme og for at reducere byrden ved re-review.